com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -37,7 +37,13 @@ public sealed class ShaderResources
             [Reload("Shaders/Utils/Blit.shader")]
             public Shader blitPS;
 
-            [Reload("Shaders/Utils/CopyDepth.shader")]
+            [SerializeField, Reload("Shaders/Utils/FullScreenDebug.shader")]
```

Comment:
What's the reason we added serialize field?

Reply:
Now in UniversalRendererData but I've also removed the "SerializeField".